README.md

```diff
@@ -1,77 +1,15 @@
 # Landing [![Build Status](https://drone.srcd.host/api/badges/src-d/landing/status.svg)](https://drone.srcd.host/src-d/landing)
 
-Serves the source{d} Landing and its API
+Serves the source{d} Landing and its API. The Landing site is also used as a [boilerplate for our documentations](https://github.com/src-d/docs).
```

Comment:
Please check: https://github.com/campoy/guide/blob/eefc9e2fa5c0b5b6f8aec6c7430e15ce846aec73/engineering/documents/README.tmpl.md

Reply:
In which sense?